### PR TITLE
Second Part of the Project

### PR DESCRIPTION
**Pull request by marurevi**
## Second Part of the Project
- Creation of two new sections
- Use of Grid and Flex
- Add images into the Images folder
- Linters

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,30 +1,141 @@
...
+      <section id="aboutMySelf" class="sectionAboutMySelf">
+        <h2>About</h2>
+        <p>At Microverse, you’ll join one of the most collaborative and social learning communities in the world.
+          Throughout the program, you'll learn through collaborating with your peers in daily meetings, group 
+          projects, and remote pair programming.
+        </p>
+        <button class="btn" type="button">Get my CV</button>
+        <div class="grid-containerAbout">
+          <div class="blue text">
+            <h3>Languages</h3>
```
Comment (smunozmo) [inline index.html:106]: - [ ] You are almost there! Just align these titles to the left just like the original template:

Template:
Captura de pantalla 2022-03-18 a la(s) 8 59 22 p  m

Your design:
Captura de pantalla 2022-03-18 a la(s) 8 59 35 p  m

File: index.html [hunk 2]
```diff
@@ -1,30 +1,141 @@
...
+        <div class="grid-containerWork">
+          <div><img src="./images/pj1.png" alt="imagen oficina 1"></div>
+          <div>
+            <h3>Project name goes here</h3>
+            <div class="tools">
+              <h4>HTML/CSS</h4>
+              <h4>Ruby on Rails</h4>
+              <h4>JavaScript</h4>
+            </div>
+            <a href="#works">See this project  <span><img src="./images/arrow.png" alt='arrow'/></span></a>
```
Comment (smunozmo) [inline index.html:43]: - [ ] Good job creating this button, however, there's should be a Hover and Pressed state in your `style.css`

Template:
Captura de pantalla 2022-03-18 a la(s) 9 04 34 p  m

**Review by smunozmo — CHANGES_REQUESTED** [review]
Hi @marurevi, 👋 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights 👏 

- Great work using the right GitHub Flow.
- Your README looks professional, well done!
- Commits are descriptive, good work.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- 

Cheers and Happy coding! 🤓 

✅  Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

⚠️  **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by marurevi** [comment]
##Html
- Fix problems with <ul> y <li> on "About me" section.
## CSS
- Remove unless borders
- Stylization buttons
- Align text according template on "About me" section.

File: index.html [hunk 3]
```diff
@@ -1,30 +1,135 @@
...
+            <a href="#works">See this project  <span><img src="./images/arrow.png" alt='arrow'/></span></a> 
+          </div>
+          <div><img src="./images/pj2.png" alt="imagen oficina 2"></div>
+          <div>
+            <h3>Project name goes here</h3>
+            <div class="tools">
+              <h4>HTML/CSS</h4>
+              <h4>Ruby on Rails</h4>
+              <h4>JavaScript</h4>
+            </div>
```
Comment (AmaduKamara) [inline index.html:52]: [ Required Changes ]

- [ ] Kindly use list items to list these technologies as requested by the previous reviewer. It's a requirement that you list them using `ul` and `li` tag.
```code
<ul class="tools">
     <li>HTML/CSS</li>
      <li>Ruby on Rails</li>
      <li>JavaScript</li>
 </ul>

File: index.html [hunk 4]
```diff
@@ -1,30 +1,135 @@
...
+            <a href="#works">See this project  <span><img src="./images/arrow.png" alt='arrow'/></span></a> 
+          </div>
+          <div><img src="./images/pj3.png" alt="imagen oficina 3"></div>
+          <div>
+            <h3>Project name goes here</h3>
+            <div class="tools">
+              <h4>HTML/CSS</h4>
+              <h4>Ruby on Rails</h4>
+              <h4>JavaScript</h4>
+            </div>
```
Comment (AmaduKamara) [inline index.html:62]: [ Required Changes ]

- [ ] Kindly use list items to list these technologies as requested by the previous reviewer. It's a requirement that you list them using `ul` and `li` tag.
```code
<ul class="tools">
     <li>HTML/CSS</li>
      <li>Ruby on Rails</li>
      <li>JavaScript</li>
 </ul>

File: index.html [hunk 5]
```diff
@@ -1,30 +1,135 @@
...
+            <a href="#works">See this project  <span><img src="./images/arrow.png" alt='arrow'/></span></a> 
+          </div>
+          <div><img src="./images/pj4.png" alt="imagen oficina 4"></div>
+          <div>
+            <h3>Project name goes here</h3>
+            <div class="tools">
+              <h4>HTML/CSS</h4>
+              <h4>Ruby on Rails</h4>
+              <h4>JavaScript</h4>
+            </div>
```
Comment (AmaduKamara) [inline index.html:72]: [ Required Changes ]

- [ ] Kindly use list items to list these technologies as requested by the previous reviewer. It's a requirement that you list them using `ul` and `li` tag.
```code
<ul class="tools">
     <li>HTML/CSS</li>
      <li>Ruby on Rails</li>
      <li>JavaScript</li>
 </ul>

File: index.html [hunk 6]
```diff
@@ -1,30 +1,135 @@
...
+            <a href="#works">See this project  <span><img src="./images/arrow.png" alt='arrow'/></span></a> 
+          </div>
+          <div><img src="./images/pj6.png" alt="imagen oficina 6"></div>
+          <div>
+            <h3>Project name goes here</h3>
+            <div class="tools">
+              <h4>HTML/CSS</h4>
+              <h4>Ruby on Rails</h4>
+              <h4>JavaScript</h4>
+            </div>
```
Comment (AmaduKamara) [inline index.html:92]: [ Required Changes ]

- [ ] Kindly use list items to list these technologies as requested by the previous reviewer. It's a requirement that you list them using `ul` and `li` tag.
```code
<ul class="tools">
     <li>HTML/CSS</li>
      <li>Ruby on Rails</li>
      <li>JavaScript</li>
 </ul>

File: style.css [hunk 7]
```diff
@@ -93,17 +132,118 @@ p {
...
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  font-family: 'Poppins', serif;
+  font-size: 20px;
+  font-style: normal;
+  font-weight: bolder;
+  line-height: 30px;
+  color: white;
+}
```
Comment (AmaduKamara) [inline style.css:239]: { Required Changes ]

- [ ] Kindly align these list items to the left to match the design given in Figma as requested by the previous reviewer.

- Your Design
image

- The Figma Design 
image

Hint: You can replace your `align-items: center;` left and add padding-left. Fix this by using these two styles.
```code
align-items: left;
padding-left: 50px;

**Review by AmaduKamara — CHANGES_REQUESTED** [review]
# Status: REQUIRED CHANGES ♻️♻️♻️
Hi @marurevi, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

SoCloseKeeganMichaelKeyGIF (2)


## Highlights
✔️ Linters are good
✔️ Good PR description
✔️ Good commit messages

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: index.html (hunk 3 above)
Comment (marurevi) [reply index.html]: I apologize for that. I fix the other section and I didn't check this one. I fix the mistakes. I hope that solves the issue.

File: style.css (hunk 7 above)
Comment (marurevi) [reply style.css]: Thanks for the hint. It was really helpful!

**Comment by marurevi** [comment]
## Html
- Fix problems with the unordered list on the "Projects" section.
## CSS
- Align text according to template on "About me" section.

**Review by medaminedev66 — APPROVED** [review]
## Hi @marurevi , 

# Status: Approved ✔️
Your project is complete!  There is nothing else to say other than... it's time to merge it 🚢
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**